src/redux/books/bookSlice.jsx

```diff
@@ -0,0 +1,21 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  booksArr: [],
+  categoriesArr: [],
```

Comment:
- [ ] This categories array shouldn't be part of your `bookSlice`. You are handling all states related to categories in your `categoriesSlice`. Please consider removing it! 🤓 🔧